akka-actor/src/main/scala/akka/util/ByteString.scala

```diff
@@ -528,7 +528,7 @@ object CompactByteString {
     else {
       val ar = new Array[Byte](bytes.size)
       bytes.copyToArray(ar)
-      CompactByteString(ar)
+      ByteString.ByteString1C(ar)
```

Comment:
Yes, it can, though it's rare case.

Unlike Java, there varargs are translated to array argument, Scala varargs are translated to Seq argument, so method like `def info(a: Int*)` have signature `(Lscala/collection/Seq;)V`.

And when casting some existing seq to vararg by `someSeq : _*` it's not copied to array, just passed as is. For example:
```
scala> def info(a: Int*) = println(a.getClass)
scala> info(Vector(1,2,3,4,5): _*)
class scala.collection.immutable.Vector
```